.github/workflows/flink_cdc_migration_test.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           submodules: true
       - uses: actions/setup-java@v4
         with:
-          java-version: 8
+          java-version: 11
```

Comment:
Do we need to add jdk 8 here?

Reply:
I'm a little unsure about how to run CI tests across Java versions. According to @GOODBOY008's opinion, it is not suggested to have too much GitHub action jobs at once, since there's a hard parallelism limit on GitHub runners (20 for free plan, 40 / 60 for paid users), and it would be expensive to have too many jobs in CI matrix.

Reply:
Added Java version matrix into migration test CI. Now there are 57 jobs to run.

Reply:
Sounds good, I'll have a try.